Loader/ComboLoader.php

```diff
@@ -162,4 +178,31 @@ private function findFileExtension($file)
     {
         return substr(strrchr($file, '_'), 1);
     }
+
+    private function isTemplateFile($file)
+    {
+        return preg_match('#/tpl/handlebars/.*_js#', $file) === 1;
+    }
+
+    private function renderTemplate($file)
+    {
+        preg_match('#/tpl/handlebars/(.*)_js#', $file, $matches);
+        $module = $matches[1];
+
+        $path = $this->getModuleSetting($module, 'path');
+
+        return $this->templating->render(
```

Comment:
created https://jira.ez.no/browse/EZP-25174, you can put this feature in the :ship: :D